electrumsv/wallet.py

```diff
@@ -230,6 +226,9 @@ def __init__(self, wallet: 'Wallet', row: AccountRow,
             if row.masterkey_id is not None)
         self._payment_requests: Dict[int, PaymentRequestRow] = {}
 
+        # { txids -> { scripthashes: [ <set of txo indices> ]} }
+        self._key_txos: Dict[ str, Dict[Script, Set[int]] ] = {}
```

Comment:
There is no key involved here, it's transaction scripts really, is there a better name than key_txos? Or is key_txos correct?

Reply:
Also, Script -> bytes?

Reply:
Changed to script_txos